test/docker.js

```diff
@@ -478,6 +480,26 @@ describe('images', function () {
         done();
       });
     });
+    it('should 404 on save image if it does not exist', function (done) {
+      docker.getImage('fake').get(function (err, res) {
+        expect(err.statusCode).to.equal(404);
+        done();
+      });
+    });
+    it('should save an image', function (done) {
+      docker.getImage('testImage').get(handleStream(done));
+    });
+    it('should load an image', function (done) {
+      var imageStream = fs.createReadStream('misc/busybox.tar');
+      docker.loadImage(imageStream, function (err) {
+        if (err) { return done(err); }
+        docker.listImages(function (err, images) {
+          if (err) { return done(err); }
+          expect(images).to.have.length(5);
```

Comment:
you're expecting `5` here - how do you know it's `5`? how do you know the one _you added_ is in that list?